go/vt/sqlparser/ast.go

```diff
@@ -227,6 +227,12 @@ type (
 	// Rollback represents a Rollback statement.
 	Rollback struct{}
 
+	// Explain represents an EXPLAIN statement
+	Explain struct {
+		Fmt       string
```

Comment:
`Format` is a method already declared on this struct, which is why I used `Fmt` instead

Reply:
may be ExplainFmt . small fmt is a format package. So looking to avoid similar name.

Reply:
I ended up naming it `Type`, because that is what the mysql docs call it - see `explain_type` here: https://dev.mysql.com/doc/refman/8.0/en/explain.html